gatsby-node.js

```diff
@@ -97,12 +103,20 @@ exports.createPages = ({ actions, graphql }) => {
             }
           }
         }
+        site {
+          siteMetadata {
+            frameworks
+          }
+        }
```

Comment:
We could also just pull this data in directly here (like you do with `docsToc`, and not actually put it in site metadata. It would entail putting the full feature support matrix on every docs page's context, but maybe that's fine?

It might be more consistent. LMK what you think @kylesuss

Reply:
I think it prob belongs on siteMetadata tbh. I moved the docsToc because I thought I couldn't have it on siteMetadata and still query for it (given the shape can change). But seems like you are querying for objects and arrays that way here and if that works, I'd say leave it on siteMetadata. The docsToc is also not something I think can live on siteMetadata anymore given it is now framework specific and so it will change page to page.

Reply:
OK!